liquibase-core/src/main/java/liquibase/resource/FileResource.java

```diff
@@ -0,0 +1,31 @@
+package liquibase.resource;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileResource extends AbstractResource {
+    private final Path file;
+
+    public FileResource(String path, File file) {
+        this(path, file.toPath());
+    }
+
+    public FileResource(String path, Path file) {
+        super(path);
+        this.file = file;
+    }
+
+    @Override
+    public InputStream openInputStream() throws IOException {
+        return Files.newInputStream(file);
+    }
+
+    @Override
+    public OutputStream openOutputStream() throws IOException {
+        return Files.newOutputStream(file, openOptions);
```

Comment:
Where do these openOptions come from? Is this still TBD?

Reply:
Those were originally passed to openOutputStream but took them out of the method signature without deleting the usage of it. I have a fix for that locally already I hadn't pushed up yet.